.github/workflows/ci-integration-db-live.yaml

```diff
@@ -38,12 +38,15 @@ jobs:
         if: ${{ steps.secret-check.outputs.available == 'true' }}
         run:
           pip install ".[integration]"
+          pip install sqlalchemy -U
       - name: Integration Test
         if: ${{ steps.secret-check.outputs.available == 'true' }}
         env:
           SF_USERNAME: ${{ secrets.SF_USERNAME }}
           SF_PASSWORD: ${{ secrets.SF_PASSWORD }}
           SF_DATABASE: ${{ secrets.SF_DATABASE }}
         run:
+          # ensure we're running sqlalchemy 2
+          python -c "import sqlalchemy; assert int(sqlalchemy.__version__.split('.')[0]) == 2"
```

Comment:
why do we need this line? If the version is < 2, should we fail it?

Reply:
Yeah, this is the same as how we did in https://github.com/ploomber/jupysql/blob/master/.github/workflows/ci.yaml#L47

Reply:
This is to make sure we are running sqlalchemy 2 on integration testing, since we don't have unit test code to test it while running. Putting the validation here should be fine?

Reply:
ok. I think we can print the version number since this might be more helpful to debug than just a boolean. @idomic what do you think?

Reply:
Gotcha